### ISSUE TITLE
missing path in verifier message for payload directory

### ISSUE THREAD

**Issue by rvanheest** [LibraryOfCongress/bagit-java#123]
In `MandatoryVerifier.checkPayloadDirectoryExists`, when the `bag/data` directory does not exist, it returns "_File [{}] should exist but it doesn't!_". The path does not seemed to be filled in correctly.

**Comment by jscancella** [comment]
@acdha line https://github.com/LibraryOfCongress/bagit-java/blob/66fc4654ff807111d89d4ee34e8bfe167056f75c/src/main/java/gov/loc/repository/bagit/verify/MandatoryVerifier.java#L87 should be 
`throw new MissingPayloadDirectoryException(messages.getString("file_should_exist_error"), dataDir);`

**Comment by rvanheest** [comment]
@jscancella shouldn't it also do something with `MessageFormatter.format`. I now see that in other places in that class as well...

**Comment by jscancella** [comment]
@rvanheest yes, sorry that's what I get for replying too quickly. I had it in my head it was the logger, not an exception.
